hotspot/src/os/linux/vm/os_linux.cpp

```diff
@@ -581,6 +580,13 @@ void os::Linux::hotspot_sigmask(Thread* thread) {
 // detecting pthread library
 
 void os::Linux::libpthread_init() {
+#if 1
+  // Hard code Alpine Linux supported musl compatible settings
```

Comment:
Is this intentional? How is this Alpine Linux specific with `#if 1`? If not Alpine specific, why hard-code to `glibc 2.9`?

Reply:
Fixed, thank you for pointing that out!